.github/workflows/test.yml

```
@@ -24,6 +24,12 @@ jobs:
           - 3.7
           - 3.8
           - 3.9
+        dev_deps:
+          - false
+        include:
+          - os: ubuntu-18.04
+            python: 3.6
```

```suggestion
            python: 3.7
```
since most likely we would jump to 3.7 as minimal in #392